src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java

```diff
@@ -0,0 +1,45 @@
+package org.fogbowcloud.app.jes.arrebol.models;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class ArrebolJob {
+
+    private static final long serialVersionUID = -6111900503095749695L;
+
+    private String id;
+
+    private ArrebolJobState jobState;
+
+    private Map<String, ArrebolTask> tasks;
+
```

Comment:
why do we need this class?

Reply:
Because is used by gson to mapped get job response from Arrebol Service.